package.json

```diff
@@ -55,14 +55,14 @@
     "@material-ui/core": "3.9.4",
     "@material-ui/styles": "4.10.0",
     "cozy-bar": "7.7.8",
-    "cozy-client": "16.0.0",
+    "cozy-client": "16.1.1",
```

Comment:
With gitup "F", it is easy to fixup commits ;)

Reply:
oublié de squasher... j'ai bien le fixup en plus... Ça m'apprendra à pas relire les commits en soumettant